package.json

```diff
@@ -82,6 +82,7 @@
     "store": "^2.0.12",
     "stream": "^0.0.2",
     "swiper": "^11.0.3",
+    "three": "^0.165.0",
```

Comment:
Why is this needed?

Reply:
this three dependency is for the creating 3D animation with WebGL